debug.log

```diff
@@ -0,0 +1 @@
+[1122/181517.624:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
This file should not exist :)

Reply:
The issue got fixed by someone else. Don't mind the PR.